Implementations/AdobeAnalytics/adobe-analytics.ts

```diff
@@ -26,14 +26,19 @@ async function submitNewCsMatchingKey() {
     csMatchingKey: csMatchingKeyValue,
     timestamp: Date.now(),
   };
+
+  console.debug({
```

Comment:
is this on purpose ?

Reply:
yeah, i put it on purpose

Clients would not use this code repo, this is just demo, and helps testing me the implementations to se if it works on Adobe dashboard